demo/liveLessons/index.html

```diff
@@ -56,8 +57,7 @@
     <script src="infra/demoShared/translate.config.js"></script>
     <script src="infra/demoShared/env.service.js"></script>
     <script src="infra/demoShared/storagesConfig.js"></script>
-    <script src="locale.run.js"></script>
+    <!--<script src="locale.run.js"></script>-->
```

Comment:
I got 404 error for this file and since I don't really use it, I decided to remove it.